alerts/cluster-monitoring-operator/PrometheusRemoteStorageFailures.md

```diff
@@ -0,0 +1,41 @@
+# PrometheusRemoteStorageFailures
+
+## Meaning
+
+The `PrometheusRemoteStorageFailures` alert triggers when there has been
+a constant increase in failures sending samples to the remote storage for more
+than 15 minutes, both in CMO Prometheus or in UWM.
```

Comment:
We don't use CMO and UWM in user-facing docs but rather "Platform Prometheus" and "user-defined Prometheus".